### ISSUE TITLE
Fetchers return byte strings

### ISSUE THREAD

**Issue by AlexandreAbraham** [nilearn/nilearn#1181]
Most of the fetchers, when they return string phenotypic data, return byte strings (in Python3). This is confusing for users that do not understand why a simple string comparison does not work. We should probably decode them before returning.

**Comment by GaelVaroquaux** [comment]
+1

**Comment by souravsingh** [comment]
I would like to work on the issue. Which functions need editing?

**Comment by KamalakerDadi** [comment]
>Which functions need editing?

Mostly, functions which returns phenotypic information are located in nilearn/datasets/func.py. You may have to look at those functions.

**Comment by AlexandreAbraham** [comment]
Actually, modifying the functions reading the CSV files should be the best approach IMHO. THey should have options to convert string automatically.

**Comment by GaelVaroquaux** [comment]
> Actually, modifying the functions reading the CSV files should be the best approach IMHO.

+1

**Comment by jeromedockes** [comment]
note: this also applies to atlas labels

**Comment by kchawla-pi** [comment]
Note: this also applies to atlas and dataset descriptions.
For example:
```python
yeo_atlas = nilearn.datasets.fetch_atlas_yeo_2011()
print(yeo_atlas['description'])
```
gives
```python
b"Yeo 2011 Atlas\n\n\nNotes\n-----\nThis atlas provides 
```
while
```python
print(yeo_atlas['description'].decode())
```
gives
```
Yeo 2011 Atlas


Notes
-----
This atlas provides a labeling of some cortical voxels in the MNI152
space.
```
I will start work on this soon. This is also a good first issue I think. If you agree I will label it as such.

**Comment by emdupre** [comment]
I think @illdopejake is working on this in https://github.com/nilearn/nilearn/pull/2037, too !

**Comment by NicolasGensollen** [comment]
Solved by #2655 and #2829
